docs/user_guide/configuring.rst

```diff
@@ -272,6 +272,25 @@ default, you can use the following configuration in ``conf.py``:
 This will make the first two navigations show up by default (AKA, top-level
 pages and their immediate children).
 
+Collapse parts / captions
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+If your pages are divided into parts, you can also hide all the pages by default
```

Comment:
Can we define what `parts/captions` means and document it? I do not see any other references to `parts` nor `captions` on the `contributing.rst` file and I think some sort of definition (how to setup parts/captions) is needed before saying that you can collapse those things.

Reply:
I agree. I think we can at least link to the document which explains parts / captions like for example the sphinx documentation, outlining toctree https://www.sphinx-doc.org/en/master/usage/restructuredtext/directives.html